modules/upgrade/pages/k-compatibility.adoc

```diff
@@ -42,7 +53,7 @@ NOTE: Beta versions are available only for testing and feedback. They are not su
 |25.1.1-beta1, 0.4.41, 0.4.36
 |25.1.1-beta1, 2.4.x, 2.3.x
 |3.12+
-|1.29.x - 1.32.x
+|1.28.x - 1.32.x{fn-k8s-compatibility}
```

Comment:
Could you also do the same for the rows below on K8s? We could also move the footnote to outside the table but leave the reference i.e. [1] here.